### DISCUSSION TITLE
Portfolio

### DISCUSSION THREAD

**Discussion by ghost** [community/community#67416] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

How about this [Repository](https://github.com/AvinashAnand02/Avinash-Portfolio) which is hosted [Portfolio](https://avinashanand.me/) isn't it good what you all say ?

**Comment by ZhengLinLei** [comment, marked as answer]
I like it 👌

**Comment by queenofcorgis** [comment]
Hi @coders06 , we’re glad you’re here 😀

Thanks for sharing your website with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
